contracts/bridges/BridgeTimelock.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../ITimelock.sol";
+
+contract BridgeTimelock is ITimelock {
+    /** Custom errors **/
+    error BadDelay();
+    error BadETA();
+    error TransactionExpired();
+    error TransactionNotQueued();
+    error TransactionNotReady();
+    error TransactionReverted();
+    error Unauthorized();
+
+    uint public immutable gracePeriod;
+    uint public immutable minimumDelay;
+    uint public immutable maximumDelay;
+
+    address public admin;
+    address public pendingAdmin;
+    uint public delay;
+
+    mapping (bytes32 => bool) public queuedTransactions;
+
+    constructor(address admin_, uint delay_, uint gracePeriod_, uint minimumDelay_, uint maxiumumDelay_) public {
+        if (delay_ < minimumDelay_) revert BadDelay();
```

Comment:
I converted all of the `require`s from the original Timelock into `revert`s with Custom Errors.

probably worth some extra scrutiny to make sure I got the logic inversions correct

Reply:
deleted this change to keep Timelock as similar as possible to its original version